css/style.css

```diff
@@ -28,17 +28,23 @@ body::before
 }
 
 /* show signature download on hover */
-span.sig_btn > a:last-child
+span.sig_btn
 {
+    position: relative;
+    height: 4.4em;
```

Comment:
> I think you meant visibility, not opacity

The thing where you don't change the layout. Apparently [`visibility`](https://css-tricks.com/almanac/properties/v/visibility/) can do that as well.